src/views/Settings.js

```diff
@@ -0,0 +1,184 @@
+import React, { useState, useCallback, useEffect } from 'react';
+import validate from 'validate.js';
+import Grid from '@material-ui/core/Grid';
+import { Typography } from '@material-ui/core';
+import { makeStyles } from '@material-ui/styles';
+import { AppButton, AppInput } from '../components';
+
+const useStyles = makeStyles({
+  root: {
+    padding: '30px',
+  },
+  heading: {
+    color: '#1c313a',
+  },
+});
+const inputText = {
+  name: 'Your name',
+  surname: 'Your family name',
+  email: 'Your email',
+  profession: 'Your profession',
+  website: 'Your website',
+  password: 'Your password',
+};
+
+const DEFAULT_FORM_STATE = {
+  isValid: false, // True when all Input Values are entered correctly
+  values: {}, // List of Input Values as string|boolean
+  touched: {}, // List of Inputs have been touched as boolean
+  errors: {}, // List of Errors for every field as array[] of strings
+};
+
+const SIGNUP_FORM_SCHEMA = {
+  nameFirst: {
+    length: {
+      minimum: 2,
+      maximum: 64,
+    },
+  },
+  nameLast: {
+    length: {
+      minimum: 2,
+      maximum: 64,
+    },
+  },
+  email: {
+    presence: { allowEmpty: false, message: 'is required' },
+    email: true,
+  },
+  // password: {
+  //   presence: { allowEmpty: false, message: 'is required' },
+  //   length: {
+  //     minimum: 8,
+  //     maximum: 32,
+  //     message: 'length between 8 and 32 chars',
+  //   },
+  // },
+  // confirmPassword: {
+  //   equality: {
+  //     attribute: 'password',
+  //     // message: 'not match',
+  //   },
+  // }
+};
+
+const Settings = () => {
+  const classes = useStyles();
+  const [formState, setFormState] = useState(DEFAULT_FORM_STATE);
```

Comment:
Adding default values to statre makes innput "managed" by defasult

`useState({...DEFAULT_FORM_STATE, ... {values.name: '', values.email: '...});`